h2o-core/src/main/java/water/api/SchemaMetadata.java

```diff
@@ -222,9 +222,27 @@ public static String consType(Schema schema, Class clz, String field_name, API a
       if (String.class.isAssignableFrom(clz))
         return "string"; // lower-case, to be less Java-centric
 
-      if (clz.equals(Boolean.TYPE) || clz.equals(Byte.TYPE) || clz.equals(Short.TYPE) || clz.equals(Integer.TYPE) || clz.equals(Long.TYPE) || clz.equals(Float.TYPE) || clz.equals(Double.TYPE))
+      if (clz.equals(Boolean.TYPE) 
+              || clz.equals(Byte.TYPE) 
+              || clz.equals(Short.TYPE) 
+              || clz.equals(Integer.TYPE) 
+              || clz.equals(Long.TYPE) 
+              || clz.equals(Float.TYPE)
+              || clz.equals(Double.TYPE))
         return clz.toString();
 
+      // use the same representation for wrapper classes as for their primitive.
```

Comment:
Where are we starting to use objects instead of primitives in the schemas?

Reply:
I can revert this now, it was a former attempt to use objects for `TargetEncoderHandler`: the problem is that the legacy implementation was using the primitive `blending` there, and in Python client for example, there was no way to specify the blending value for the `transform` method, which means that `blending` was always `false`, even if the model was trained with `blending=true`...
There were several inconsistencies like that that were making the `transform` API especially very hard to understand.

I solved this default boolean value issue (how to identify if a boolean parameter was provided by user or not?), by setting the default value on client side, except for Flow, that doesn't allow to use the `transform` method: actually I'm even wondering if TE is fully usable on Flow, given that an encoder is supposed be applied differently on a train frame and a test or validation frame...